src/components/Comments.jsx

```diff
@@ -16,7 +15,7 @@ export function Comments({ reviewId }) {
         console.error(e);
         window.alert("ERROR: Could not contact the server, try again later");
       });
-  }, [isActive]);
+  });
```

Comment:
removing the dependency array is problematic here - it means your component runs the effect on every re-render.
When deleting a comment, it looks quite jarring as the comments are fetched again. Also, when you type anything in the text box you set state (which you should for a controlled component) and causes another fetch. Just put in an empty array here